paddle/scripts/travis/build_doc.sh

```diff
@@ -35,6 +28,7 @@ TARGET_BRANCH="gh-pages"
 SOURCE_BRANCH="master"
 
 # Clone the repo to output directory
+mkdir output
```

Comment:
Should we use `rm -rf output` here?  It seems that `rmdir output` works only when output is an empty directory.